addons/website/models/website.py

```diff
@@ -1723,7 +1723,9 @@ def _search_get_indirect_fields(self, fields, model):
         for field in fields:
             if '.' not in field:
                 continue
-            direct, indirect = field.split('.')
+            direct, indirect = field.split('.', 1)
+            if '.' in indirect:
+                direct, indirect = indirect.split('.')
```

Comment:
The traceback will occur again if someone adds more than 2 fields.

Example- `res['search_fields'].extend(['channel_id.tag_ids.group_id.name'`

Reply:
> The traceback will occur again if someone adds more than 2 fields.
> 
> Example- `res['search_fields'].extend(['channel_id.tag_ids.group_id.name'`

Yes.
It will only works for some case and if we add more than 2 fields we gets the trace back again.
so for that kind of case we can make some change in website_helpdesk_slides module.

Thankyou so much 👍